int32 id
float32[] camera_center
float32[] marker_center
float32[] marker_size

